app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+    belongs_to :post
+    after_save :update_comment_counter
+
+    def update_comment_counter
+        post.increment!(:comments_counter)
+      end
```

Comment:
- [ ] As mentioned by the former reviewer, kindly make the `update_comment_counter` a private method. This will ensure that outside methods can't update the `comments_counter` but done automatically through this method. 👍 💻